relinker/src/main/java/com/getkeepsafe/relinker/TextUtils.java

```diff
@@ -1,32 +0,0 @@
-/**
- * Copyright 2015 - 2016 KeepSafe Software, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.getkeepsafe.relinker;
-
-/**
- * Slimmed down version of https://developer.android.com/reference/android/text/TextUtils.html to
- * avoid depending on android.text.TextUtils.
```

Comment:
Why delete this now?  As the comment here shows, we had explicitly decided _not_ to use android.text.TextUtils.  Has something changed here?